modules/microshift-embed-microshift-update-osbuilder-worker.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-offline-deploy.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-microshift-update-osbuilder-worker_{context}"]
+= Updating osbuilder worker configuration to prepare for image building
+
+After you have updated the blueprint, you must update the osbuilder worker configuration to prepare for building the image with embedded {microshift-short} containers.
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have created an `/etc/osbuild-worker/osbuild-worker.toml` configuration file if does not exist.
```

Comment:
I'm not sure it should be a prerequisite. We're creating or updating this file in the current section. 
There is no need to "pre-create" it.

Reply:
It's part of the procedure, not prerequisite.

Reply:
@ggiguash This comes from your own doc that says "you might need to create this file." We are only updating it, not creating it.